app/models/project.rb

```diff
@@ -192,6 +194,26 @@ def extra_answer(num)
     (answer = read_attribute("extra_answer_#{num}".to_sym)) && answer.present? ? answer : nil
   end
 
+  def hide!(reason, user)
+    update_attributes(
+      hidden_reason: reason,
+      hidden_by_user_id: user.id,
+      hidden_at: Time.now
```

Comment:
We should use Time.zone.now so the time stored will be correct regardless of what the system time zone is set as. 

http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/